File: Programming Languages/Homework 7/three.py
# Prompt the user for input
k = input('Enter an integer between 1 and 8: ')

def fun1():
    j = 2 * k - 1
    print ('j: ' + str(j))

def fun2():
    j = 3 * k + 1
    print ('j: ' + str(j))

def fun3():
    j = 4 * k - 1
    print ('j: ' + str(j))

def fun4():
    j = k - 2
    print ('j: ' + str(j))

options = {
    1: fun1,
    2: fun1,
    3: fun2,
    4: fun3,
    5: fun2,
    6: fun4,
    7: fun4,
    8: fun4
}

options[k]()

File: Programming Languages/Homework 7/three.cpp
#include <iostream>
using namespace std;

int main() {
    int i, j, k;

    // Prompt user for variable k
    cout << "Enter an integer between 1 and 8: ";
    cin >> k;

    switch(k) {
        case (1): case (2):
            j = 2 * k - 1;
            break;
        case (3): case (5):
            j = 3 * k + 1;
            break;
        case (4):
            j = 4 * k - 1;
            break;
        case (6): case (7): case (8):
            j = k - 2;
            break;
        default:
            cout << "You did not inpupt an integer between 1 and 8" << endl;
    }

    cout << "j: " << j << endl;
}
